django/contrib/auth/hashers.py

```diff
@@ -172,6 +173,7 @@ class BasePasswordHasher:
     """
     algorithm = None
     library = None
+    salt_entropy = 128
```

Comment:
Please change entropy to `71`. This cleanup should not change the current behavior.

Reply:
Updated.